### PR TITLE
REP-100 Add documentation

### PR DESCRIPTION
**Pull request by edwh**
Added a README in a suitable format.  This passes validation at https://wordpress.org/plugins/developers/readme-validator/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,39 @@
...
+2. Click *Add New*
+3. Select the ZIP file
+4. Activate the plugin
+
+Once installed and activated, the plugin can be used on a WordPress page using the `repair-directory` shortcode, e.g.
+
+`[repair-directory region="Wales" width="100%" height="500" region="London" domain="https://map.restarters.net"`
+
+The following options are available:
+* **domain** default `https://map.restarters.net`.  This is the URL of your domain, which is including the Repair
```
Comment (ngm) [inline README.md]: Presume this should be the full URL of the embedding page, not just domain, e.g. https://therestartproject.org/repairdirectory ?

**Review by ngm — COMMENTED** [review]
Question about the info on the domain parameter.

File: README.md (hunk 1 above)
Comment (edwh) [reply README.md]: Yes.  Pushed a change to clarify.

Comment (ngm) [reply README.md]: Thanks - actually I was thinking about the path.  Given that the parameter is called domain, people might then think this means just the domain.  But presumably this parameter also allows the inclusion of a path?

So for example on trp.org the parameter would be e.g.:

```
domain="https://therestartproject.org/repairdirectory"
```

I wonder if the parameter name should be changed to 'url'.

**Pull request merged**